### PR TITLE
FIX: Do not run STC if SliceTiming metadata is set but empty

### PR DESCRIPTION
**Pull request by oesteban**
Currently, the BIDS-Validator would just raise a Warning when metadata contains:

```
{
  "SliceTiming": []
}
```

### CODE REVIEW

File: fmriprep/workflows/bold/base.py [hunk 1]
```diff
@@ -357,7 +357,7 @@ def init_func_preproc_wf(
             fmaps.append({'suffix': 'syn'})
 
         # Short circuits: (True and True and (False or 'TooShort')) == 'TooShort'
-        run_stc = ("SliceTiming" in metadata and
+        run_stc = (metadata.get("SliceTiming", False) and
```
Comment (effigies) [inline fmriprep/workflows/bold/base.py]: If `None` doesn't work, `[]` isn't going to work.
```suggestion
        run_stc = (bool(metadata.get("SliceTiming")) and
```

**Comment by effigies** [comment]
Have you tested this on a dataset?

**Comment by oesteban** [comment]
Just submitted a job on lonestar5 to run ds001242, which has this problem. Will report back when done.

**Comment by oesteban** [comment]
The fix is working out. Merging :)

**Pull request merged**